examples/simple.py

```diff
@@ -41,16 +44,16 @@ def main(data_url_1: str, data_url_2: str, result_url: str):
     _dummy_write(result_url, result_data)
     get_tracking_context().write_mode = WriteMode.OVERWRITE
 
-    print('DONE')
+    logging.info('DONE')
```

Comment:
Is the `root_logger` prefered way of logging? 

Or is there a reason why not to use logger object for logging? 

```
import logging

logger = logging.getLogger()
logger.setLevel(logging.INFO)
```

Reply:
Thanks for spotting it. I was using logging incorrectly. Fixed!